File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.cc
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    // 통화 목록 개수 및 통화 목록 입력 받기
    int cnt = 0;

    cin >> cnt;

    vector<int> call_list(cnt, 0);
    
    for(int i = 0; i < cnt; i++) cin >> call_list[i];


    // 영식 민식 통화 요금 구하기
    int y = 0;
    int m = 0;

    for(int i = 0; i < cnt; i++)
    {
        int num = call_list[i];
        
        while(num >= 0)
        {
            num-=30;
            y += 10;
        }

        num = call_list[i];

        while(num >= 0)
        {
            num-=60;
            m += 15;
        }
    }

    // 영식 요금이 더 저렴할 경우
    if( y < m ) cout << "Y" << " " << y;

    // 민식 요금이 더 저렴할 경우
    else if( m < y ) cout << "M" << " " << m;

    // 가격이 동일할 경우
    else cout << "Y" << " M " << y;
}

File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {

            // 통화 목록 갯수 입력받기
            int cnt = int.Parse(Console.ReadLine());

            // 통화 목록 값 입력 받기
            var call_List = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();

            // 영식, 민식 요금 구하기
            int y = 0;
            int m = 0;
            
            for(int i = 0; i < cnt; i++)
            {
                int num = call_List[i];

                while(num >= 0)
                {
                    num -= 30;
                    y += 10;
                }

                num = call_List[i];

                while(num >= 0)
                {
                    num -= 60;
                    m += 15;
                }
            }

            // 요금제 비교 후 출력
            if(y < m) Console.WriteLine($"Y {y}");

            else if(m < y) Console.WriteLine($"M {m}");
            
            else Console.WriteLine($"Y M {y}");
            
        }
    }
}